lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -211,8 +212,8 @@ class IndexQueryTestCase(ModuleStoreTestCase):
 
     @override_waffle_flag(CONTENT_TYPE_GATING_FLAG, True)
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 10, 157),
-        (ModuleStoreEnum.Type.split, 4, 153),
+        (ModuleStoreEnum.Type.mongo, 10, 160),
```

Comment:
It seems like this add 3 SQL queries across the board. Do we know which ones? Should we be concerned about them? Could we cache them?

Reply:
i'm assuming the extra queries are to check the feature flag and get the user_course_expiration date.  
we could probably cache them? as with the previous PR where this came up, should adding caching be a part of this PR?

Reply:
I think it's fine to leave it as is, just want to make sure we're thinking about it.